network/src/components/AppWrapper.js

```diff
@@ -646,8 +724,13 @@ export const AppWrapper = ({children}) => {
             }}>
             <div className="button-container">
               <button class='addPerson' onClick={() => {
-                  setPersonDialog(true);      // Open the "Add Person" dialog
-                  resetCanvasContextMenu();   // Reset the canvas context menu
+                  setPersonDialog(true);     
+                  resetCanvasContextMenu();  
+                  }} 
+                  style={{ padding: '10px 20px', 
+                    margin: '5px', 
+                    backgroundColor: 'green', 
+                    color: 'white'
```

Comment:
In 0.0.2, this does not exist. These changes were made in PR #27. Before this branch can be merged, these changes need to be brought into parody. In your local branch, run `git pull` then `git merge 0.0.2`. You may need to resolve some conflicts after that. Once those changes are resolved, commit & push your local branch.

Reply:
Updated